tests/yamls/multi-deployment-crd.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
```

Comment:
where this file is used?

Reply:
It'll be called in `roxctl_verification.sh`, which will be called within [run.sh](https://github.com/stackrox/stackrox/blob/6787c87a8ec2f2f409b4cd80a4ca581d73e871e0/tests/e2e/run.sh#L138).
I added these tests to the list of refactoring to bats within [ROX-8801](https://issues.redhat.com/browse/ROX-8801) so we don't forget about them.

Reply:
So maybe add a comment in the file

Reply:
Added TODO marker within `roxctl_verification.sh` as well as comment header for all YAML files which resources they contain + that they are being used wtihin `roxctl_verification.sh`.